cmd/configwiz/configwiz/printer.go

```diff
@@ -0,0 +1,62 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package configwiz
+
+import (
+	"fmt"
+	"strings"
+)
+
+func printLine(s string) {
+	fmt.Print(s)
+}
+
+type indentingPrinter2 struct {
```

Comment:
Can we rename this to `indentingPrinter` and remove the current `indentingPrinter`?